main.md

```diff
@@ -275,6 +275,121 @@ claim values), SD-JWT Salt/Value Containers (containing the mapping of the
 plain-text claim values and the salt values), and SD-JWT Releases (containing a
 subset of the same mapping).
 
+## The Challenge of Canonicalization
+
+When receiving an SD-JWT with an associated Release, a verifier must be able to
+derive all claim values that are to be disclosed (in cleartext) and must be able
+to check that each claim value's hash was signed by the issuer.
+
+Usually, JSON-based formats transport claim values as simple properties of a JSON object such as this:
+
+```
+...
+  "family_name": "Möbius",
+  "address": {
+    "street_address": "Schulstr. 12",
+    "locality": "Schulpforta"
+  }
+...
+```
+
+However, a problem arises when signatures over the data need to be computed and
+verified. Common signature schemes require the same byte string as input to the
+signature verification as was used for creating the signature. In the salted
```

Comment:
can this be generalized to `a problem arises when computation over the data need to be performed and verified` to cover signing and hashing from the onset?

Reply:
Yes, sounds good to me.